addons/website/views/snippets/s_numbers_list.xml

```diff
@@ -9,50 +9,50 @@
                 </div>
-                <div class="col-lg-6 offset-lg-1 s_col_no_bgcolor" data-name="Numbers group">
-                    <div class="row">
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">15%</span>
-                            <p>Revenue Growth</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">$50M</span>
-                            <p>Expected revenue</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">85%</span>
-                            <p>User Retention</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">100,000</span>
-                            <p>Website visitors</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">20+</span>
-                            <p>Projects deployed</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
-                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
-                            <span class="h2-fs">4x</span>
-                            <p>Inventory turnover</p>
-                            <div class="s_hr pt16 pb40">
-                                <hr class="w-100 mx-auto"/>
-                            </div>
-                        </div>
+                <div class="col-lg-3 offset-lg-1">
+                    <p>
+                        <span class="h2-fs">15%</span><br/>
+                        Revenue Growth
+                    </p>
+                    <div class="s_hr pt16 pb40">
+                        <hr class="w-100 mx-auto"/>
+                    </div>
```

Comment:
This is missing the data-snippet. Should be fixed in 18.0 in its own PR.

Reply:
Here is the PR, I added `data-name` too

https://github.com/odoo/odoo/pull/185897